composer.json

```
@@ -19,7 +19,7 @@
         "symfony/yaml"  : "^3.4"
     },
     "require-dev" : {
-        "friendsofphp/php-cs-fixer": "^2.18"
+        "friendsofphp/php-cs-fixer": "2.18"
```

```suggestion
        "friendsofphp/php-cs-fixer": "2.18.0"
```

IMO the patch version also should be specified. Patch versions often "fix existing fixers" and so they break existing CI because a fixer "suddenly starts working more correctly" and now finds more edge-cases of something (which it should have found in the first place, but the fixer was deficient). So the "fix for the fixer" really is a patch, but of course it is not strictly backward-compatible.